README.md

```diff
@@ -77,6 +77,7 @@ optional arguments:
                         request; increases runtime; requires Tor to be
                         installed and in system path.
   --csv                 Create Comma-Separated Values (CSV) File.
+  --xlsx                Create the standard file for the modern Microsoft Excel spreadsheet (xlsx).
```

Comment:
xlsx file option can be used and therefore it should be written in the README.md as well.